com.github.lainsce.timetable.yml

```diff
@@ -0,0 +1,35 @@
+app-id: com.github.lainsce.timetable
+base: io.elementary.BaseApp
+base-version: juno
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '18.08'
+command: com.github.lainsce.timetable
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=ca.desrt.dconf
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --filesystem=host
```

Comment:
Having host access is not that good. The app needs that just for saving/opening files? if so, the app should use Gtk.NativeFileChoose instead of FileChooser and remove that host filesystem access otherwise the app won't be sandboxed at all.

Reply:
Is `Gtk.NativeFileChoose` a thing on Gtk 3.22 or not? This app mainly targets Juno and not GNOME.

Reply:
It's there since 3.20 https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html
```
a new GtkFileChooserNative

Since: 3.20
```